README.md

```diff
@@ -6,8 +6,7 @@ and blog. The site is [built][contributing.md] using [Hugo][] and hosted on
 
 ## Get involved
 
-If you are new to OpenTelemetry and just get started with it, you are in a
-perfect position to help us get better: the website and documentation is the
+If you are new to OpenTelemetry and just get started with it, you are in a perfect position to help us get better: the website and documentation is the
```

Comment:
⚠️ This is seeding a formatting issue to see if Prettier will cause the build to fail.

Reply:
This got detected as expected.